### DISCUSSION TITLE
Fav skins

### DISCUSSION THREAD

**Discussion by Pop06** [ppy/osu#24014] [Ideas] [answered]
It will be cool to add a fav tab for skins that you dont have scavage through skins to find your fav skin that you like or looking for
![Screenshot 2023-06-23 093101](https://github.com/ppy/osu/assets/137274065/4632b4df-657d-417b-803e-4049bcafa6ed)

**Comment by soopax** [comment, marked as answer]
This has been requested before in https://github.com/ppy/osu/discussions/23534.
